src/server.coffee

```diff
@@ -31,6 +34,9 @@ class Server
       else if '--port' == @argv[0]
         @argv.shift()
         @options.port = @argv.shift() | 0
+      else if '--test-path' == @argv[0]
+        @argv.shift()
+        @options.testPath = @argv.shift() | 'test/mock-api'
```

Comment:
The pipe on line 33 above converts the string to a number.  I don't think it will create a default here the way you expect.